### ISSUE TITLE
Add support for Deno

### ISSUE THREAD

**Issue by Olian04** [Olian04/gleam_stdin#2]
```
$ cat README.md | gleam run --target=javascript --runtime=deno
   Compiled in 0.01s
    Running win_test.main
error: Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'stdin')
  process.stdin.resume();
          ^
    at stdinGenerator (file:///C:/Users/rift9/projects/win_test/build/dev/javascript/stdin/js_ffi.mjs:7:11)
    at stdinGenerator.next (<anonymous>)
    at read_line (file:///C:/Users/rift9/projects/win_test/build/dev/javascript/stdin/js_ffi.mjs:48:22)
    at read_line (file:///C:/Users/rift9/projects/win_test/build/dev/javascript/stdin/stdin.mjs:8:15)
    at file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:102:53
    at file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:84:13
    at file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:472:13
    at file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:247:13
    at do_fold (file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:152:13)
    at fold (file:///C:/Users/rift9/projects/win_test/build/dev/javascript/gleam_stdlib/gleam/iterator.mjs:167:10)
```

**Comment by Olian04** [comment]
It's now working for Deno as well

```
$ cat README.md | gleam test --target=javascript --runtime=deno
   Compiled in 0.01s
    Running stdin_test.main
["[![Package Version](https://img.shields.io/hexpm/v/stdin)](https://hex.pm/packages/stdin)\r\n", "[![Hex Docs](https://img.shields.io/badge/hex-docs-ffaff3)](https://hexdocs.pm/stdin/)\r\n", "\r\n", "# stdin\r\n", "\r\n", "`stdin` provides a synchronous iterator for consuming stdin. It supports all the non-browser targets, Erlang, Node, ~~Deno~~ (WIP), and Bun.\r\n", "\r\n", "```sh\r\n", "gleam add stdin\r\n", "```\r\n", "\r\n", "```gleam\r\n", "import gleam/io\r\n", "import gleam/iterator\r\n", "import stdin.{stdin}\r\n", "\r\n", "pub fn main() {\r\n", "  stdin()\r\n", "  |> iterator.to_list\r\n", "  |> io.debug\r\n", "}\r\n", "```\r\n", "\r\n", "Further documentation can be found at <https://hexdocs.pm/stdin>.\r\n", "\r\n", "## Development\r\n", "\r\n", "```sh\r\n", "gleam run   # Run the project\r\n", "```\r\n"]

$ cat README.md | gleam test --target=javascript --runtime=node
   Compiled in 0.01s
    Running stdin_test.main
["[![Package Version](https://img.shields.io/hexpm/v/stdin)](https://hex.pm/packages/stdin)\r\n", "[![Hex Docs](https://img.shields.io/badge/hex-docs-ffaff3)](https://hexdocs.pm/stdin/)\r\n", "\r\n", "# stdin\r\n", "\r\n", "`stdin` provides a synchronous iterator for consuming stdin. It supports all the non-browser targets, Erlang, Node, ~~Deno~~ (WIP), and Bun.\r\n", "\r\n", "```sh\r\n", "gleam add stdin\r\n", "```\r\n", "\r\n", "```gleam\r\n", "import gleam/io\r\n", "import gleam/iterator\r\n", "import stdin.{stdin}\r\n", "\r\n", "pub fn main() {\r\n", "  stdin()\r\n", "  |> iterator.to_list\r\n", "  |> io.debug\r\n", "}\r\n", "```\r\n", "\r\n", "Further documentation can be found at <https://hexdocs.pm/stdin>.\r\n", "\r\n", "## Development\r\n", "\r\n", "```sh\r\n", "gleam run   # Run the project\r\n", "```\r\n"]

$ cat README.md | gleam test --target=javascript --runtime=bun
   Compiled in 0.01s
    Running stdin_test.main
["[![Package Version](https://img.shields.io/hexpm/v/stdin)](https://hex.pm/packages/stdin)\r\n", "[![Hex Docs](https://img.shields.io/badge/hex-docs-ffaff3)](https://hexdocs.pm/stdin/)\r\n", "\r\n", "# stdin\r\n", "\r\n", "`stdin` provides a synchronous iterator for consuming stdin. It supports all the non-browser targets, Erlang, Node, ~~Deno~~ (WIP), and Bun.\r\n", "\r\n", "```sh\r\n", "gleam add stdin\r\n", "```\r\n", "\r\n", "```gleam\r\n", "import gleam/io\r\n", "import gleam/iterator\r\n", "import stdin.{stdin}\r\n", "\r\n", "pub fn main() {\r\n", "  stdin()\r\n", "  |> iterator.to_list\r\n", "  |> io.debug\r\n", "}\r\n", "```\r\n", "\r\n", "Further documentation can be found at <https://hexdocs.pm/stdin>.\r\n", "\r\n", "## Development\r\n", "\r\n", "```sh\r\n", "gleam run   # Run the project\r\n", "```\r\n"]

$ cat README.md | gleam test --target=erlang
   Compiled in 0.01s
    Running stdin_test.main
["[![Package Version](https://img.shields.io/hexpm/v/stdin)](https://hex.pm/packages/stdin)\n", "[![Hex Docs](https://img.shields.io/badge/hex-docs-ffaff3)](https://hexdocs.pm/stdin/)\n", "\n", "# stdin\n", "\n", "`stdin` provides a synchronous iterator for consuming stdin. It supports all the non-browser targets, Erlang, Node, ~~Deno~~ (WIP), and Bun.\n", "\n", "```sh\n", "gleam add stdin\n", "```\n", "\n", "```gleam\n", "import gleam/io\n", "import gleam/iterator\n", "import stdin.{stdin}\n", "\n", "pub fn main() {\n", "  stdin()\n", "  |> iterator.to_list\n", "  |> io.debug\n", "}\n", "```\n", "\n", "Further documentation can be found at <https://hexdocs.pm/stdin>.\n", "\n", "## Development\n", "\n", "```sh\n", "gleam run   # Run the project\n", "```\n"]
```

**Comment by Olian04** [comment]
Fixed as of [version 1.1.2](https://github.com/Olian04/gleam-stdin/tree/v1.1.2)
